conf/routes

```diff
@@ -12,4 +12,5 @@ GET     /message                    controllers.AsyncController.message
 # Map static resources from the /public folder to the /assets URL path
 GET     /assets/*file               controllers.Assets.versioned(file)
 
-->     /helloworld.GreeterService   controllers.GreeterRouter
\ No newline at end of file
+->     /helloworld.GreeterService   io.grpc.examples.helloworld.GreeterServiceRouter
+->     /echo.EchoService     example.myapp.echo.grpc.EchoServiceRouter
```

Comment:
I'm unfamiliar with the -> syntax, that's a new one that we added for grpc/akka routes or was it always there? :-)

Reply:
That was already there, for forwarding a whole prefix to a Router

Reply:
I see, cool!